idl/control-plane-agent.idol

```diff
@@ -69,6 +69,14 @@ Interface(
             reply: Simple("UartClient"),
             idempotent: true,
         ),
+        "get_installinator_image_id": (
+            doc: "Get the current installinator image ID; writes 0 bytes if we have no ID.",
```

Comment:
Nit: "writes 0 bytes" is confusing (is it writing no bytes, or bytes with the value 0?)

How about "does not modify the leased buffer" or "fills the buffer with 0s" respectively?

Reply:
Good call; reworded to

> Get the current installinator image ID, returning the number of bytes written. If we have no installinator image ID, does not modify the leased buffer and returns 0.